### PR TITLE
Update Itemdb (Ragcan's JRO item)

### PR DESCRIPTION
**Pull request by attackjom**
* **Addressed Issue(s)**:  -

* **Server Mode**:  Renewal

* **Description of Pull Request**:  

- Implement bLongSPGainvalue  and bLongHPGainvalue
- Add  new RC2 that use in itemscript
- Ragcan's JRO itemdb 

* Info take from 
- manual translate from JRO/IRO page of each item https://www.divine-pride.net/
- Sigma's translation in divine-pride forum (changelog section)

### CODE REVIEW

**Comment by MrLekkz** [comment]
Thanks @attackjom
I found one mistake: Ladon's Skin (20949) should increase hit by 5 and not by 15. 
Apart from that, everything seems fine.
GJ!

**Comment by attackjom** [comment]
> Thanks @attackjom
> I found one mistake: Ladon's Skin (20949) should increase hit by 5 and not by 15.
> Apart from that, everything seems fine.
> GJ!

done.

File: db/re/item_combo_db.txt [hunk 1]
```diff
@@ -349,6 +379,19 @@
 2990:2991,{ bonus bAllStats,5; bonus5 bAutoSpellWhenHit,"SC_MAELSTROM",1,100,BF_MAGIC,0; }
 2995:15110,{ bonus2 bSkillAtk,"NC_AXEBOOMERANG",100; }
 2997:2999,{ bonus2 bAddClass,Class_All,10; bonus bMatkRate,10; }
+32242:18971,{ bonus2 bFixedCastrate,"RK_STORMBLAST",-100; }
+32242:18973,{ bonus2 bSkillAtk,"NC_FLAMELAUNCHER",70; bonus2 bSkillAtk,"NC_COLDSLOWER",70; }
```
Comment (MrLekkz) [inline db/re/item_combo_db.txt]: on iRO, Old Detachments Ring has a set bonus with Old Mitra (18972) (50% hp bonus)
32242:18972,{ bonus bMaxHPrate,50; }
can't confirm if its iRO exclusive or not

Comment (attackjom) [reply db/re/item_combo_db.txt]: It seem I miss that haha.

Comment (attackjom) [reply db/re/item_combo_db.txt]: done

File: db/re/item_db.txt [hunk 2]
```diff
@@ -12066,6 +12180,12 @@
...
+28430,Arch_Bishop_Ring,Arch Bishop Ring,4,20,,400,,0,,1,0xFFFFFFFF,63,2,136,,90,0,0,{ bonus bLuk,2; bonus bFlee2,5; bonus bUseSPrate,-5; bonus bHealPower,15; bonus bAddItemHealRate,15; },{},{}
+28433,Luminous_Blue_Stone,Luminous Blue Stone,4,20,,400,,0,,1,0xFFFFFFFF,63,2,136,,90,0,0,{ bonus bAgi,2; bonus bCritical,5; bonus bAspdRate,5; bonus bCritAtkRate,2; skill "MG_SIGHT",1; if (readparam(bLuk)>=100) { bonus bCritical,10; bonus bCritAtkRate,10; } },{},{}
+28434,Fist_Fighter_Glove,Fist Fighter's Glove,4,20,,400,,0,,1,0xFFFFFFFF,63,2,136,,90,0,0,{ bonus bStr,2; bonus bMaxSP,100; bonus2 bAddClass,Class_All,2; skill "MO_ABSORBSPIRITS",1; if (readparam(bStr)>99) bonus2 bAddClass,Class_All,5; },{},{}
+28437,Hibram's_Gloves,Hibram's Gloves,4,20,,400,,0,,1,0xFFFFFFFF,63,2,136,,90,0,0,{ skill "AL_HEAL",3; bonus bInt,2; bonus bMatkRate,2; bonus bUseSPrate,-5; if (readparam(bInt>=100)) bonus bMatkRate,5; },{},{}
```
Comment (Angelic234) [inline db/re/item_db.txt]: if (readparam(bInt>=100))  should be if (readparam(bInt)>=100)

Comment (attackjom) [reply db/re/item_db.txt]: done

**Comment by Atemo** [comment]
hmm why did you close the PR?

**Comment by attackjom** [comment]
Mistake in editing master branch
so, I decide to remake fork

**Pull request closed without merging**